pages/mesosphere/dcos/services/cassandra/2.7.0-3.11.4/updates/index.md

```diff
@@ -9,3 +9,11 @@ render: mustache
 ---
 
 #include /mesosphere/dcos/services/include/update.tmpl
+
+# Upgrading directly from 2.5.0-3.11.3
```

Comment:
Do we need this section? My comment was more to make this note in the release notes because we had similar instructions in the previous release notes (`2.6.0-3.11.4`). I don't think we need this as an individual page.

Reply:
Thanks, Sam! My thinking is that a user will naturally look for update information on the update page, which is a standard part of the Cassandra document suite. While it is true that the 2.6.0 release notes include the same information, there are no actual 2.6.0 *documents* any longer. This means that the section you added can be found ONLY in the Release Notes. It's good documentation practice to make information, especially shortcuts, available wherever a user might look. That's why we are going to leave the update tip in *both* the Release Notes and the Update page.